timemachine/md/minimizer.py

```diff
@@ -509,20 +581,31 @@ def local_minimize(
         Restraint k to wrap val_and_grad_fn in a positional harmonic restraint to the input positions of the local_idxs.
         If None, minimize with no positional restraint. Refer to `timemachine.potentials.bonded.harmonic_positional_restraint`
         for implementation.
+
     Returns
     -------
     Optimized set of coordinates (N,3)
 
+    Raises
+    ------
+    AssertionError
+    MinimizationError
+
     """
 
+    if type(minimizer_config) not in (FireMinimizationConfig, ScipyMinimizationConfig):
```

Comment:
nit: I think `isinstance` is generally preferred to `type(...) in ...`

```suggestion
    if not isinstance(minimizer_config, MinimizationConfig):
```

Reply:
Don't know what I was thinking...